cli/features/tutorial.feature

```diff
@@ -134,7 +134,9 @@ Feature: Following the tutorial
           - 'mongo'
       """
     When running "exo setup" in this application's directory
+    And setting up a temporary mongo docker container
```

Comment:
Great, thanks! How much sense does this PR with #81 in place? Can we do #81 instead of this PR?